lgc/patch/llpcSystemValues.cpp

```diff
@@ -41,11 +41,12 @@
 using namespace lgc;
 using namespace llvm;
 
+// -enable-shadow-desc: enable shadow descriptor table
```

Comment:
enable-shadow-desc -> enable-shadow-desc-table
Try to keep the option's name consist with the option variable name